app/src/ui/preferences/advanced.tsx

```diff
@@ -203,25 +218,81 @@ export class Advanced extends React.Component<
         />
         <p className="git-settings-description">
           Allows the display of notifications when high-signal events take place
-          in the current repository.{this.renderNotificationSettingsLink()}
+          in the current repository.{this.renderNotificationHint()}
         </p>
       </div>
     )
   }
 
-  private renderNotificationSettingsLink() {
-    if (!__DARWIN__ && !isWindows10OrLater()) {
+  private onGrantNotificationPermission = async () => {
+    await requestNotificationsPermission()
+    this.updateNotificationsState()
+  }
+
+  private async updateNotificationsState() {
+    const notificationsPermission = await getNotificationsPermission()
+    this.setState({
+      suggestGrantNotificationPermission:
+        supportsNotificationsPermissionRequest() &&
+        notificationsPermission === 'default',
+      warnNotificationsDenied: notificationsPermission === 'denied',
+      suggestConfigureNotifications: notificationsPermission === 'granted',
+    })
+  }
+
+  private renderNotificationHint() {
+    // No need to bother the user if their environment doesn't support our
+    // notifications or if they've been explicitly disabled.
+    if (!supportsNotifications() || !this.props.notificationsEnabled) {
+      return null
+    }
+
+    const {
+      suggestGrantNotificationPermission,
+      warnNotificationsDenied,
+      suggestConfigureNotifications,
+    } = this.state
+
+    if (suggestGrantNotificationPermission) {
+      return (
+        <>
+          {' '}
+          You need to{' '}
+          <LinkButton onClick={this.onGrantNotificationPermission}>
+            grant permission
+          </LinkButton>{' '}
+          to display these notifications from GitHub Desktop.
+        </>
+      )
+    }
+
+    const notificationSettingsURL = getNotificationSettingsUrl()
+
+    if (notificationSettingsURL === null) {
       return null
     }
 
-    const notificationSettingsURL = __DARWIN__
-      ? 'x-apple.systempreferences:com.apple.preference.notifications'
-      : 'ms-settings:notifications'
+    if (warnNotificationsDenied) {
+      return (
+        <p>
```

Comment:
Console is angry at this. :D 
![image](https://user-images.githubusercontent.com/75402236/168647263-ee396493-43b4-4e0f-9b3f-088d15ee83e2.png)

Reply:
Fixed in e532ecf614c504e94de0c35c35fc6d22695cab2a